web.go

```diff
@@ -30,5 +30,15 @@ func poweredBy(res http.ResponseWriter, req *http.Request) {
 	}
 	// Print the string to the ResponseWriter
 	hostname, _ := os.Hostname()
-	fmt.Fprintf(res, "Powered by %v\nRelease %v on %v\n", powered, release, hostname)
+	fmt.Fprintf(w, "Powered by %v\nRelease %v on %v\n", powered, release, hostname)
+}
+
+// Log the HTTP request
+func logHandler(w http.ResponseWriter, r *http.Request) {
+	log.Printf("%s %s %s", r.RemoteAddr, r.Method, r.URL)
+}
+
+func rootHandler(w http.ResponseWriter, r *http.Request) {
+	logHandler(w, r)
```

Comment:
If we're trying to display best practices for logging requests, we should be handling this via middleware:

```
func logRequestMiddleware(h http.Handler) http.Handler {
  return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
    log.Printf("%s %s %s", r.RemoteAddr, r.Method, r.URL)
    // delegate request to the next handler
    h.ServeHTTP(w, r)
  })
}

// back in main
http.HandleFunc("/", logRequestMiddleware(http.HandlerFunc(poweredByHandler)))
```